### PR TITLE
Repos housekeeping

### PR DESCRIPTION
**Pull request by Karolk99**

### CODE REVIEW

File: lib/membrane_webrtc_plugin/sdp.ex [hunk 1]
```diff
@@ -132,7 +131,7 @@ defmodule Membrane.WebRTC.SDP do
 
     track.rids
     |> Enum.reduce(media, fn rid, media ->
-      Media.add_attribute(media, "rid:#{rid} recv")
+      Media.add_attribute(media, "rid:#{inspect(rid)} recv")
```
Comment (daniel-jodlos) [inline lib/membrane_webrtc_plugin/sdp.ex:134]: Isn't rid a binary? If it is, we probably don't need to invite `Inspect` to this party

File: lib/membrane_webrtc_plugin/track/constraints.ex [hunk 2]
```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.WebRTC.Track.Constraints do
   @moduledoc false
 
-  alias ExSDP.{FMTP, RTPMapping}
+  alias ExSDP.Attribute.{FMTP, RTPMapping}
```
Comment (daniel-jodlos) [inline lib/membrane_webrtc_plugin/track/constraints.ex:4]: Outstanding. How did it even compile previously? 🤣  Good to know there are no silly errors in our most important product...

**Review by daniel-jodlos — APPROVED** [review]
Looks good, 12/10 👍🏻

File: lib/membrane_webrtc_plugin/sdp.ex (hunk 1 above)
Comment (Karolk99) [reply lib/membrane_webrtc_plugin/sdp.ex:134]: rid can also be nil and because of that dialyzer throws error

**Pull request merged**